File: Armstrong.py
 al = input("Sayı Giriniz: ")
  bul = len(str(al))
  say = 0
  for i in range(bul):
    say = say + int(al[i]) ** int(bul)
  if say == al:
    print("{} armstrong sayısıdır".format(al))
  else:
    print("{} armstrong sayısı değildir.".format(al))

File: Armstrong.java
import java.util.Scanner;


public class Main {
    public static void main(String[] args) {
        
        Scanner scanner = new Scanner(System.in);
        
        System.out.print("Sayıyı Giriniz: ");
        int sayi = scanner.nextInt();
        System.out.print("Basamak Sayısı : ");
        int basamak_sayisi = scanner.nextInt();
        
        int gecici_sayi = sayi;
        int toplam = 0;
        
        do {
            int basamak_degeri = gecici_sayi % 10;
            gecici_sayi /= 10;
            
            toplam += Math.pow(basamak_degeri,basamak_sayisi);
            
        }while(gecici_sayi > 0);
        
        if (sayi == toplam) {
            
            System.out.println("Bu sayı bir armstrong sayısıdır...");
        }
        else {
            System.out.println("Bu sayı bir armstrong sayısı değildir...");
        }
        
        
        
    }
}
